gc-print.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Growth Chart Print Preview</title>
+    <link rel="stylesheet" type="text/css" href="css/gc-pview.css" />
+    <link rel="stylesheet" type="text/css" href="css/print.css" />
+    <link rel="stylesheet" type="text/css" href="css/gc-pview-print.css" />
+</head>
+<body>
+<div id="print-toolbar">
+    <a href="javascript:window.print();" class="print-link">
+        <img src="img/print.png" alt="" />
+        <span data-translatecontent="STR_6020"></span>
+    </a>
+</div>
+<div id="container">
+    <div id="header">
+        <h1 class="patient-name" data-translatecontent="STR_6021">Patient`s name</h1>
```

Comment:
This code was carried forward from the existing print window. 
Checked with @kolkheang and wont be making any change for this as we want to retain whatever we can from the existing print functionality.